lldb/test/API/lang/swift/stdlib/ContiguousArray/TestContiguousArray.py

```diff
@@ -20,9 +20,6 @@ def test_frame_contiguous_array(self):
         lldbutil.run_to_source_breakpoint(
             self, 'Set breakpoint here', lldb.SBFileSpec('main.swift'))
 
-        self.expect("frame variable --dynamic-type run-target",
```

Comment:
note: I removed `--dynamic-type run-target`, it seems not relevant, and isn't required.